src/main/java/org/folio/repository/holdings/DbHolding.java

```diff
@@ -12,6 +12,7 @@
 )
 @Getter
 @Setter
+@EqualsAndHashCode
```

Comment:
```
@ToString
```
it's always nice to have toString() method also

Reply:
thanks, will add